### PR TITLE
[PREVIEW] cnp azure vault

### PR DESCRIPTION
**Pull request by david-gimelle**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: infrastructure/main.tf [hunk 1]
```diff
@@ -90,20 +90,16 @@ provider "vault" {
...
-  name = "${local.vaultName}"
+  name                = "${local.vaultName}"
   resource_group_name = "${local.vaultName}"
 }
 
-data "vault_generic_secret" "idam_secret" {
-  path = "secret/${var.vault_section}/ccidam/idam-api/oauth2/client-secrets/divorce"
-}
-
-data "azurerm_key_vault_secret" "session_secret" {
```
Comment (tchow8) [inline infrastructure/main.tf]: I think you accidently deleted the session_secret?

File: infrastructure/main.tf [hunk 2]
```diff
@@ -85,25 +85,22 @@ module "frontend" {
   }
 }
 
-provider "vault" {
```
Comment (tchow8) [inline infrastructure/main.tf]: Please add this back in

Comment (tchow8) [reply infrastructure/main.tf]: Ignore please.

**Comment by nsingamsetty** [comment]
Test is completed and compared both AAT and preview environment and the keys are not changed against AAT. the only change i see is below keys and it is acceptable after discussion with Tony. PR merged successfully and marking the ticket to Done.

REDIS_ENCRYPTION_SECRET
APPINSIGHTS_INSTRUMENTATIONKEY

**Pull request merged**